lib/refinery/products.rb

```diff
@@ -10,6 +10,9 @@ module Products
     require 'refinery/products/engine'
     require 'refinery/products/configuration'
 
+    require 'refinery/products/categories/url'
+    require 'refinery/products/categories/finder'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.